st2common/st2common/util/param.py

```diff
@@ -150,8 +152,32 @@ def _render(node, render_context):
     Render the node depending on its type
     '''
     if 'template' in node:
+        complex_type = False
+
+        LOG.debug("Rendering param of type: %s", type(node['template']))
```

Comment:
Never mind, I see you do log the whole template couple if lines below.

I'm fine with that, although it could potentially cause problems if template contains sensitive data (although this shouldn't be an issue since it's a DEBUG log level).